4-task/store/bookstore.go

```diff
@@ -0,0 +1,25 @@
+package store
+
+import "awesomeProject1/entity"
+
+type Book interface {
+	GetAll() []entity.Book
```

Comment:
let's add here an error too (to be able to test it) and you will understand in the future why we did it
```suggestion
	GetAll() ([]entity.Book, error)
```

Reply:
because of the change you are going to make some changes to handler and tests, and add one more test case, but it is worth it